besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -360,6 +364,8 @@ public class BesuCommand implements DefaultCommandValues, Runnable {
   @CommandLine.ArgGroup(validate = false, heading = "@|bold P2P Discovery Options|@%n")
   P2PDiscoveryOptionGroup p2PDiscoveryOptionGroup = new P2PDiscoveryOptionGroup();
 
+  private final TransactionSelectionServiceImpl transactionSelectionServiceImpl;
```

Comment:
I think it's a good idea to annotate it with @`Unstable` as we have for the other new services

Reply:
Done